src/common/io/tls/client.c

```diff
@@ -288,48 +287,64 @@ tlsClientOpen(THIS_VOID)
         if (verifyResult != X509_V_OK)                                                                              // {vm_covered}
         {
             THROW_FMT(                                                                                              // {vm_covered}
-                CryptoError, "unable to verify certificate presented by '%s:%u': [%ld] %s",                         // {vm_covered}
-                strZ(sckClientHost(this->socketClient)), sckClientPort(this->socketClient), verifyResult,           // {vm_covered}
-                X509_verify_cert_error_string(verifyResult));                                                       // {vm_covered}
+                CryptoError, "unable to verify certificate presented by '%s': [%ld] %s",                            // {vm_covered}
+                strZ(ioClientName(this->ioClient)), verifyResult, X509_verify_cert_error_string(verifyResult));     // {vm_covered}
         }
 
         // Verify that the hostname appears in the certificate
         X509 *certificate = SSL_get_peer_certificate(session);                                                      // {vm_covered}
-        bool nameResult = tlsClientHostVerify(sckClientHost(this->socketClient), certificate);                      // {vm_covered}
+        bool nameResult = tlsClientHostVerify(this->host, certificate);                                             // {vm_covered}
         X509_free(certificate);                                                                                     // {vm_covered}
 
         if (!nameResult)                                                                                            // {vm_covered}
         {
             THROW_FMT(                                                                                              // {vm_covered}
                 CryptoError,                                                                                        // {vm_covered}
                 "unable to find hostname '%s' in certificate common name or subject alternative names",             // {vm_covered}
-                strZ(sckClientHost(this->socketClient)));                                                           // {vm_covered}
+                strZ(this->host));                                                                                  // {vm_covered}
         }
     }
 
     FUNCTION_LOG_RETURN(IO_SESSION, result);
 }
 
+/**********************************************************************************************************************************/
+static const String *
+tlsClientName(THIS_VOID)
+{
+    THIS(TlsClient);
+
+    FUNCTION_TEST_BEGIN();
+        FUNCTION_TEST_PARAM(TLS_CLIENT, this);
+    FUNCTION_TEST_END();
+
+    ASSERT(this != NULL);
+
+    FUNCTION_TEST_RETURN(ioClientName(this->ioClient));
```

Comment:
Not sure I entirely get what the purpose of the "name" construct is...   If it's for identifying information, I would think you'd want to add a "TLS: " or something to it instead of just passing it through as-is from the underlying layer...

Reply:
We already have the type that gets output in logging as `tls` so no need to repeat it. The name is just some identifier for the ioClient. When the ioClient is a socket this will be `host:port`.

Reply:
ok, makes sense.